supervisor/api/auth.py

```diff
@@ -67,11 +67,17 @@ async def auth(self, request: web.Request) -> bool:
 
         # Json
         if request.headers.get(CONTENT_TYPE) == CONTENT_TYPE_JSON:
+            if request.method == "GET":
+                raise HTTPBadRequest(text="json auth requires HTTP POST instead of GET")
             data = await request.json()
             return await self._process_dict(request, addon, data)
 
         # URL encoded
         if request.headers.get(CONTENT_TYPE) == CONTENT_TYPE_URL:
+            if request.method == "GET":
+                raise HTTPBadRequest(
+                    text="URL encoded auth requires HTTP POST instead of GET"
+                )
```

Comment:
Of curse, that is correct, but do we need to guide people there? We can still just accept GET without future validation if GET or POST.

Reply:
I removed those checks. It becomes a one line pull request :-)